andino_hardware/README.md

```diff
@@ -21,36 +19,37 @@ This package aims to provide the necessary information to the correct assembly o
 
 ## Assembly Process
 
-Following you can see a photo with all the parts:
+The steps to reproduce the mounting of the Andino are:
```

Comment:
Being super obnoxious (nit ^10) :nerd_face:  : I guess we can remove this line as the section and steps are self explanatory